app/controllers/api/v1/users_controller.rb

```diff
@@ -1,4 +1,8 @@
 class Api::V1::UsersController < ApplicationController
+  def index
```

Comment:
gives Sara an opportunity to differentiate different User models -- build out serializer before upload to heroku

Reply:
user_role ; what else?

Reply:
worker_responsibilities; business -- more pending with time